submissions/ArthurGorbenko/frogger-game/js/app.js

```diff
@@ -0,0 +1,130 @@
+let countForWonRound = 0;
+
+const blockWidth = 100;
+const blockHeight = 90;
+const fieldMaxHeight = 400;
+const fieldMinHeight = -70;
+const fieldMaxWidth = 405;
+const fieldMinWidth = -10;
+const waterLevel = fieldMinHeight + blockHeight;
+const startPositionX = 200;
+const startPositionY = 300;
+
+class Enemy {
+  constructor(x, y) {
+    this.sprite = "images/enemy-bug.png";
+    this.x = x;
+    this.y = y;
+    this.speed = Math.floor(Math.random() * 100) + 50;
+  }
+  update(dt) {
+    this.x += this.speed * dt;
+
+    if (this.x > 600) {
+      this.x = -100;
+      this.generateSpeed(countForWonRound);
+    }
+
+    const halfOfSizeImg = 50;
```

Comment:
This value will never change, but the constant is created over and over again on every method call. Let's put it outside `update` method.

Reply:
Done.